lib/hydra/works/models/concerns/generic_file/characterization.rb

```diff
@@ -0,0 +1,95 @@
+include RDF
+
+module Hydra::Works::GenericFile
+  module Characterization
+    extend ActiveSupport::Concern
+
+    autoload :SaxDoc, 'hydra/works/models/generic_file/characterization/sax_doc'
+
+    DEFAULT_FITS_TO_RDF = { filename: "filename",
+                            mime_type: ["identity", "toolname=FITS", "mimetype"],
+                            file_size: "size",
+                            well_formed: "well-formed",
+                            valid: "valid",
+                            last_modified: ["lastmodified", "toolname=Exiftool"],
+                            date_created: ["created", "toolname=Exiftool"],
+                            fits_version: %w(fits version),
```

Comment:
Why does this one get a %w and the others are just arrays? (I like arrays, not sure what the standards say).